packages/twenty-server/src/database/typeorm/metadata/migrations/1711104284380-addRemoteServerTable.ts

```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddRemoteServerTable1711104284380 implements MigrationInterface {
+  name = 'AddRemoteServerTable1711104284380';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `CREATE TABLE "metadata"."remoteServer" ("id" uuid NOT NULL DEFAULT uuid_generate_v4(), "fdwId" uuid NOT NULL DEFAULT uuid_generate_v4(), "fdwType" character varying, "fdwOptions" jsonb, "userMappingOptions" jsonb, "workspaceId" uuid NOT NULL, "createdAt" TIMESTAMP NOT NULL DEFAULT now(), "updatedAt" TIMESTAMP NOT NULL DEFAULT now(), CONSTRAINT "PK_8e5d208498fa2c9710bb934023a" PRIMARY KEY ("id"))`,
```

Comment:
comment disappeared but still missing FK?

Reply:
Argh sorry I didn't pay attention to this. Makes total sense I should have seen it. I'm sorry you lost time on this because of me